build/yaml/crd/nsx.vmware.com_staticroutes.yaml

```diff
@@ -0,0 +1,222 @@
+              routes:
+                description: Specify next hop routes for network
+                items:
+                  properties:
+                    destination:
+                      description: Specify network address in CIDR format
+                      format: cidr
+                      type: string
+                    name:
+                      description: Unique route name to indicate the specific node
+                        in specific TKG
+                      type: string
+                    target:
+                      description: Next hop gateway IP address
+                      format: ip
+                      type: string
+                  required:
+                  - destination
+                  - name
+                  - target
+                  type: object
+                type: array
+            required:
+            - routes
+            type: object
+          status:
+            description: StaticRouteStatus defines the observed state of StaticRoute
+            properties:
+              conditions:
+                items:
+                  description: StaticRouteCondition defines condition of StaticRoute.
+                  properties:
+                    message:
+                      description: Message shows a human readable message about condition.
+                      type: string
+                    reason:
+                      description: Reason shows a brief reason of condition.
+                      type: string
+                    status:
+                      description: Status defines status of condition type, True or
+                        False.
+                      type: string
+                    type:
+                      description: Type defines condition type.
+                      type: string
+                  required:
+                  - status
+                  - type
+                  type: object
+                type: array
+              routes:
+                items:
+                  properties:
+                    destination:
+                      description: Specify network address in CIDR format
+                      format: cidr
+                      type: string
+                    name:
+                      description: Unique route name to indicate the specific node
+                        in specific TKG
+                      type: string
+                    target:
+                      description: Next hop gateway IP address
+                      format: ip
+                      type: string
+                  required:
+                  - destination
+                  - name
+                  - target
+                  type: object
+                type: array
+            required:
+            - conditions
+            - routes
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+  - name: v1alpha1
```

Comment:
You write it twice?

Reply:
there were two ._types.go under different directories, so snippets appeared twice.